docs/pages/installation/docker-extension.md

```diff
@@ -82,7 +82,7 @@ Finally, you can also install the Meshery Docker Extension using the Docker CLI.
 
 It runs as a set of one or more containers inside your Docker Desktop virtual machine.
 
-## Remove Meshery Docker Extension
+## Remove Meshery as a Docker Extension
```

Comment:
Normally, this makes sense. Understand, though, that the software is called "Meshery Docker Extension". Either way works. 👍